### PR TITLE
Lift `previewBody` state from `FirstCommentWelcome`

### PR DESCRIPTION
**Pull request by georgeblahblah**
## What does this change?

- Lifts `previewBody` state from `FirstCommentWelcome` to `Comments`

Tested on [CODE](https://riffraff.gutools.co.uk/deployment/view/f89d05de-e0ee-466d-98ce-b39c6594b661) and seems to work!

Closes https://github.com/guardian/dotcom-rendering/issues/10224

### CODE REVIEW

File: dotcom-rendering/src/components/Discussion/CommentContainer.test.tsx [hunk 1]
```diff
@@ -74,6 +74,8 @@ describe('CommentContainer', () => {
 				setError={() => {}}
 				userNameMissing={false}
 				setUserNameMissing={() => {}}
+				previewBody=""
+				setPreviewBody={() => undefined}
```
Comment (mxdvl) [inline dotcom-rendering/src/components/Discussion/CommentContainer.test.tsx:78]: Small consistency note, other props use the `() => {}` pattern

File: dotcom-rendering/src/components/Discussion/FirstCommentWelcome.stories.tsx [hunk 2]
```diff
@@ -6,9 +6,9 @@ export default { title: 'Discussion/FirstCommentWelcome' };
 
 export const defaultStory = () => (
 	<FirstCommentWelcome
-		body="My first message ever!!"
 		submitForm={() => Promise.resolve()}
 		cancelSubmit={() => undefined}
+		previewBody=""
```
Comment (mxdvl) [inline dotcom-rendering/src/components/Discussion/FirstCommentWelcome.stories.tsx:11]: Should this not be “My first message ever!!”?

**Review by mxdvl — APPROVED** [review]
This PR is much Welcome!

File: dotcom-rendering/src/components/Discussion/CommentContainer.test.tsx (hunk 1 above)
Comment (georgeblahblah) [reply dotcom-rendering/src/components/Discussion/CommentContainer.test.tsx:78]: copied this from somewhere else where the pattern is `()=>undefined` 🙅 . I'll keep it consistent in this file!

**Pull request merged**